pkg/router/router.go

```diff
@@ -1042,7 +1073,7 @@ func (r *router) removeRouteGroupOfRule(rule routing.Rule) {
 func (r *router) checkIfTransportAvalailable() (ok bool) {
 	r.tm.WalkTransports(func(tp *transport.ManagedTransport) bool {
 		ok = true
-		return true
+		return ok
```

Comment:
We can just remove `ok = true` and `return true`.

Reply:
regarding assignment to `ok` it is in the response parameter name, since we cannot return any value from `WalkTransport`

Reply:
So why we assign OK as return value at all? I think non-assign return value and just its type is better.

Reply:
@mrpalide this piece of code is not really important btw, also I'm not sure if there's any point in returning just the type, since it will mean we still assign bool value to a new variable which we will return, will be 2 more LoC.